WhosAGoodBoy/Who's A Good Boy/Assets/Scripts/Extensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class Extensions{
+
+    public static Collider2D GetClosest(this Collider2D[] colliders, Vector3 relativeToHere)
```

Comment:
This is neat, are there limits to what you can write extension functions for in C#?

Reply:
I don't think so. Any class can be extended